cla_backend/apps/legalaid/migrations/0021_auto_20190515_1042.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import django.utils.timezone
+import model_utils.fields
+import uuidfield.fields
+import core.cloning
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("legalaid", "0020_fill_missing_outcome_codes")]
+
+    operations = [
+        migrations.CreateModel(
+            name="ContactResearchMethod",
+            fields=[
+                ("id", models.AutoField(verbose_name="ID", serialize=False, auto_created=True, primary_key=True)),
+                (
+                    "created",
+                    model_utils.fields.AutoCreatedField(
+                        default=django.utils.timezone.now, verbose_name="created", editable=False
+                    ),
+                ),
+                (
+                    "modified",
+                    model_utils.fields.AutoLastModifiedField(
+                        default=django.utils.timezone.now, verbose_name="modified", editable=False
+                    ),
+                ),
+                (
+                    "method",
+                    models.CharField(
+                        default=b"PHONE",
+                        max_length=10,
+                        null=True,
+                        blank=True,
+                        choices=[(b"EMAIL", b"Email"), (b"PHONE", b"Phone"), (b"SMS", b"Sms")],
+                    ),
+                ),
+                ("reference", uuidfield.fields.UUIDField(unique=True, max_length=32, editable=False, blank=True)),
+            ],
+            options={"abstract": False},
+            bases=(core.cloning.CloneModelMixin, models.Model),
+        ),
+        migrations.AddField(
+            model_name="personaldetails",
+            name="contact_for_research_methods",
+            field=models.ManyToManyField(to="legalaid.ContactResearchMethod", null=True),
```

Comment:
_curious_
Is this going to be an array field?

Reply:
@sldblog Yep it's moving from a single value for contact preference to multiple values

Reply:
Sorry, what I meant is that I'm curious how this column type looks in PostgreSQL.

Reply:
_not a blocker!_

Reply:
m2m through table